src/incipit/processing.py

```diff
@@ -328,15 +360,56 @@ def detect_regions_from_image(
     # Dilate to combine adjacent text contours
     kernel = cv2.getStructuringElement(
         shape=cv2.MORPH_RECT,
-        ksize=(9, 9))
-    dilate = cv2.dilate(
+        ksize=(3, 3))
+    dilateStart = cv2.dilate(
```

Comment:
@Allan518 There are several variable naming styles:
- `snake_case` (because the words are strung along with underscores `_` and the words are like humps, where a snake has eaten something and is digesting it, see Petit Prince illustration below!);
- `camelCase`, where the words are Capitalized to separate them (no underscores here or anything else)
- and in languages that allow dashes in token names (like CSS) `kebab-case`.
You can see other examples here: https://en.wikipedia.org/wiki/Naming_convention_(programming)#Examples_of_multiple-word_identifier_formats

The main rule is to be consistent. In Python, the standard is to use `snake_case`. Not only that, but in this project, specifically, all the other variables are in `snake_case`. So, as a programmer contributing code, you must absolutely do like in Rome, that is, follow the conventions that you see implicitly established.

So your `dilateStart` should be a `dilate_start`.

OK. So, that's a long rant about conventions, but at least, well done on using variable names to document the use of the variable. 😊


![2717719c2acfbed8868838c91f826e3b](https://user-images.githubusercontent.com/11770469/130306045-7d0f37a2-8b1e-44c1-bd09-c9adf313a24f.jpg)

Reply:
@jlumbroso I have made the change as suggested, please review.